st2common/st2common/bootstrap/aliasesregistrar.py

```diff
@@ -191,7 +191,8 @@ def register_aliases(packs_base_paths=None, pack_dir=None, use_pack_cache=True,
                      fail_on_failure=False):
 
     if packs_base_paths:
-        assert isinstance(packs_base_paths, list)
+        if not isinstance(packs_base_paths, list):
+            raise TypeError('The pack base paths has a value that is not a list.')
```

Comment:
Same here - help the user debug what happened by including what was the type of `packs_base_path` in the exception message.